raster/r.surf.fractal/testsuite/test_r_surf_fractal.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+
+"""
+MODULE:    Test of r.surf.fractal
+
+AUTHOR(S): Shashank Shekhar Singh <shashankshekharsingh1205 gmail com>
+
+PURPOSE:   Test fractal surface generation with r.surf.fractal module
+
+COPYRIGHT: (C) 2024 by Shashank Shekhar Singh and the GRASS Development Team
+
+This program is free software under the GNU General Public
+License (>=v2). Read the file COPYING that comes with GRASS
+for details.
+"""
+
+import os
+
+import grass.script as gs
+
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+
+
+class FractalTestCase(TestCase):
+    """Test case for r.surf.fractal"""
+
+    # Raster map name to be used as output
+    output = "fractal_result"
```

Comment:
Better use tempname: `output = gs.core.tempname(10)`

Reply:
For tests, we actually don't use the temporary names, the tests are usually ran in throw-away mapsets. So you can keep this.